infrastructure/kube/keep-test/monitoring/monitoring-ingress.yaml

```diff
@@ -0,0 +1,44 @@
+---
+## Nginx Ingress
+## Follow https://devopscube.com/setup-ingress-kubernetes-nginx-controller/
+apiVersion: networking.k8s.io/v1
+kind: Ingress
```

Comment:
Is ingress enough to expose the service publicly or is some additional GCP configuration needed?